packages/web-components/src/components/callout-with-media/__stories__/callout-with-media.stories.react.tsx

```diff
@@ -0,0 +1,91 @@
+import DDSCalloutWithMediaImage from '@carbon/ibmdotcom-web-components/es/components-react/callout-with-media/callout-with-media-image';
+import DDSCalloutWithMediaVideo from '@carbon/ibmdotcom-web-components/es/components-react/callout-with-media/callout-with-media-video';
+
+// eslint-disable-next-line sort-imports,import/no-duplicates
+import { CONTENT_BLOCK_COPY_SIZE } from '../../content-block/content-block-copy';
+import imgLg16x9 from '../../../../../storybook-images/assets/720/fpo--16x9--720x405--005.jpg';
+import readme from './README.stories.react.mdx';
+import textNullable from '../../../../.storybook/knob-text-nullable';
+
+const image = (
+  <DDSCalloutWithMediaImage
+    alt="Image alt text"
+    default-src={imgLg16x9}
+    heading="Lorem ipsum dolor sit amet, consectetur adipiscing elit."></DDSCalloutWithMediaImage>
+);
+
+const video = <DDSCalloutWithMediaVideo video-id="1_9h94wo6b"></DDSCalloutWithMediaVideo>;
+
+export const Default = ({ parameters }) => {
+  const { copy, heading, mediaType } = parameters?.props?.CalloutWithMedia ?? {};
+  const headingComponent = document.querySelector('dds-content-block-heading');
+
+  if (headingComponent) {
+    headingComponent.shadowRoot!.textContent = heading;
+  }
+
+  return (
+    <DDSCalloutWithMedia>
+      <DDSContentBlockHeading>{heading}</DDSContentBlockHeading>
+      <DDSCalloutWithMediaCopy size={CONTENT_BLOCK_COPY_SIZE.SMALL}>{copy}</DDSCalloutWithMediaCopy>
+      {mediaType === 'image' ? image : ``} {mediaType === 'video' ? video : ``}
+    </DDSCalloutWithMedia>
+  );
+};
+
+export default {
+  title: 'Components/Callout with media',
+  decorators: [
+    story => (
+      <div className="bx--grid">
+        <div className="bx--row">
+          <div className="bx--col-lg-12 bx--no-gutter">{story()}</div>
+        </div>
+      </div>
+    ),
+  ],
+  parameters: {
+    ...readme.parameters,
+    hasStoryPadding: true,
+    knobs: {
+      CalloutWithMedia: ({ groupId }) => ({
+        mediaType: select('mediaType (optional)', ['image', 'video', 'none'], 'image', groupId),
+        heading: textNullable('Heading', 'Curabitur malesuada varius mi eu posuere', groupId),
+        copy: `Lorem ipsum *dolor* sit amet, consectetur adipiscing elit. Aenean et ultricies est.
+          Mauris iaculis eget dolor nec hendrerit. Phasellus at elit sollicitudin, sodales
+          nulla quis, *consequat* libero. Here are
+          some common categories:`,
+      }),
+    },
+    propsSet: {
+      default: {
+        CalloutWithMedia: {
+          mediaType: 'image',
+          heading: 'Curabitur malesuada varius mi eu posuere',
+          copy: `Lorem ipsum *dolor* sit amet, consectetur adipiscing elit. Aenean et ultricies est.
+            Mauris iaculis eget dolor nec hendrerit. Phasellus at elit sollicitudin, sodales
+            nulla quis, *consequat* libero. Here are
+            some common categories:`,
+        },
+      },
+    },
```

Comment:
This shouldn't be necessary for the react wrappers. This is for karma testing and we don't test the wrapper stories i believe
```suggestion
    propsSet: {
      default: {
        CalloutWithMedia: {
          mediaType: 'image',
          heading: 'Curabitur malesuada varius mi eu posuere',
          copy: `Lorem ipsum *dolor* sit amet, consectetur adipiscing elit. Aenean et ultricies est.
            Mauris iaculis eget dolor nec hendrerit. Phasellus at elit sollicitudin, sodales
            nulla quis, *consequat* libero. Here are
            some common categories:`,
        },
      },
    },
```